.gitignore

```diff
@@ -1,4 +1,5 @@
 public/
 resources/
 node_modules/
-.hugo_build.lock
\ No newline at end of file
+.hugo_build.lock
+.vscode
```

Comment:
I don't see why there should be a .vscode file in the doc directory unless you added the documentation to VS code (but why should you do this?)

Reply:
umm... I've been editing all the .md files in VS Code. Was I not supposed
to do it this way?

Reply:
Oh, I wasn't aware you were regularly using VS Code other than for testing how to import the test suites etc. 
In that case that's fine :)

Reply:
Reminder of what we just talked about: please add links to these guides here: load-testing/quick-start/30-demo-test-suite/#importing-the-_posters_-test-suite-into-eclipse (you might replace the note saying "Users of other IDEs have to carry out similar steps" by links to your guides to the other IDEs).